src/app/components/match/match.component.html

```diff
@@ -1,115 +1,178 @@
+                                <small>לתיאום יש לפנות ל- {{request.contact.name}}, {{request.contact.phone}}, {{request.contact.email}}</small> 
+                            </div>
+                        </div>
+                    </div>                                                      
+                </li>
+            </ul>
+        </div>
+        <div class="col-2 d-flex align-items-center justify-content-center">
+            <button class="btn btn-primary mb-0 shadow-none" (click)="openMatchFormModal()">שידוך</button>
+        </div>
+        <div class="col-5">
+            <ul id="accordionO" class="list-group p-0 mx-0">
+                <li *ngFor="let offer of volunteeringOffers; index as i" class="list-group-item align-items-start p-0 m-0">
+                    <div class="card p-0 m-0">
+                        <div (click)="onSelectOffer(offer)" [ngClass]="{'is-selected': selectedOffer===offer}" 
+                            class="row card-header py-0 mx-0" 
+                            aria-expanded="true" [attr.aria-controls]="'#collapseO'+i" [attr.id]="'headingO'+i">
+                           <div class="col-1 d-flex align-items-center justify-content-center">
+                                <i class="fas fa-plus-circle fa-2x pl-1" data-toggle="collapse" [attr.data-target]="'#collapseO'+i"></i>
+                            </div>                            
+                            <div class="col-11">
+                                <div class="row"><h5>{{offer.organization}}</h5></div>
+                                <div class="row"><div class="card-subtitle">מעוניין לבצע התנדבות על <b>{{offer.content.title}}</b></div></div>
+                            </div>
+                        </div>                            
+                        <div [attr.id]="'collapseO'+i" class="collapse" [attr.aria-labelledby]="'headingO'+i">
+                            <div class="card-body m-1 py-0">
+                                <div class="mb-1 d-flex justify-content-between">מספר מתנדבים: {{offer.numberOfVolunteers}}
+                                    <div>רב פעמי: <i class="far fa-check-square" *ngIf="offer.multiOccurrence"></i>
+                                                <i class="far fa-square" *ngIf="!offer.multiOccurrence"></i>
+                                    </div>
+                                </div>
+                                <div><span class="badge badge-success ml-1 mb-1" *ngFor="let region of offer.regions;">{{region.name}}</span></div>
+                                <small>לתיאום יש לפנות ל- {{offer.contact.name}}, {{offer.contact.phone}}, {{offer.contact.email}}</small> 
+                            </div>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </div>
+    </div>
+    <div class="row">
+        <div class="title-divider-round"></div>
```

Comment:
What is it for? It looks like a page-control element, not like a separator.

Reply:
Haven't found something more suitable for section divide.

Reply:
`<hr/>`?